src/components/ui/sortbutton.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useRef } from 'react';
+import { View, Text, TouchableOpacity, Animated, Easing } from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+import { cn } from '../../lib/utils';
+
+export interface SortButtonProps {
+  containerClassName?: string;
+  buttonClassName?: string;
+  optionsClassName?: string;
+}
+
+interface SortOptionProps {
+  option: string;
+  isSelected: boolean;
+  onPress: () => void;
+  isLast?: boolean;
```

Comment:
remove this, and set a fixed size instead so it does not exceed the space allocated